virt/install/uninstalling-virt.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-virt"]
+include::_attributes/common-attributes.adoc[]
+= Uninstalling {VirtProductName}
+:context: uninstalling-virt
+
+toc::[]
+
+You can uninstall {VirtProductName} by using the {product-title} xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console] or the command line interface (xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[CLI]).
+
+[IMPORTANT]
+====
+You must delete all xref:../../virt/virtual_machines/virt-delete-vms.adoc#virt-delete-vm-web_virt-delete-vms[virtual machines], xref:../../virt/virtual_machines/virt-manage-vmis.adoc#virt-deleting-vmis-cli_virt-manage-vmis[virtual machine instances],
+and xref:../../virt/virtual_machines/virtual_disks/virt-deleting-datavolumes.adoc#virt-deleting-dvs_virt-deleting-datavolumes[data volumes] before you uninstall {VirtProductName}. If {VirtProductName} workloads exist, you cannot delete the `HyperConverged` custom resource (CR).
+====
+
+[id='uninstalling-virt-web-console_{context}']
+== Uninstalling {VirtProductName} using the web console
+
+You uninstall {VirtProductName} by deleting the `HyperConverged` CR, the {VirtProductName} Operator, and the `openshift-cnv` namespace.
```

Comment:
Because the user needs to perform these tasks in sequence, I would suggest rewriting this as:
````
You uninstall {VirtProductName} by performing the following tasks:

1. Delete the HyperConverged custom resource (link to procedure module)
2. Delete the {VirtProductName} Operator (link to procedure module)
3. Delete the `openshift-cnv` namespace (link to procedure module)
````

Also see my comment for lines 24 and 28

Reply:
There is also another (optional?) step:
The OLM is not going to remove CRDs which are actually cluster scoped resources so neither removing the namespace will do.
In order to get rid of leftovers, product CRDs should be manually deleted.

Reply:
@tiraboschi I happened to see this [kubevirt-dev chat thread](https://groups.google.com/g/kubevirt-dev/c/68ae5_DIT-Q/m/8BtFvugqAwAJ) when I was looking for info on how to delete CRDs. Do I need to add the step that Chandler recommends?

Reply:
also in this case you can simply query them by label:
```
$ oc delete crd --dry-run=client -l operators.coreos.com/kubevirt-hyperconverged.openshift-cnv
customresourcedefinition.apiextensions.k8s.io "cdis.cdi.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "hostpathprovisioners.hostpathprovisioner.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "hyperconvergeds.hco.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "kubevirts.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "networkaddonsconfigs.networkaddonsoperator.network.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "ssps.ssp.kubevirt.io" deleted (dry run)
customresourcedefinition.apiextensions.k8s.io "tektontasks.tektontasks.kubevirt.io" deleted (dry run)
```
remove the `--dry-run=client` option to make it effective.

Reply:
@tiraboschi The procedure is clear. Can the user do this only with the CLI? Or is there a way to do it in the UI?